### PR TITLE
Addressed issues and added capabilities

### PR DESCRIPTION
**Pull request by B1t0n**
- Added capability to upload via command line utilities. This is done by saving a key to the browser's local storage upon a successful login. Users can supply the Authorization header with the key to upload files. Also added a green button to the UI, under 'Upload', so you can copy ready made cURL commands with your key included. Now you can also force the mime-type you would like to set through HTTP header - x-pwndrop-content-type.
- Upgraded included dependencies versions as well as Go version. This fixes the autocert fails issues when requesting the server with a domain.
- Removed unnecessary installation scripts, see readme update section.

### CODE REVIEW

File: www/pages/components/file_view.vue.js [hunk 1]
```diff
@@ -702,6 +721,15 @@ var appFileView = Vue.component("app-file-view", {
 				t.isDragging = true;
 			}
         });
+
+        this.mainBus.$on('loggedIn', (username) => {
+            console.log("LoggedIN");
+            console.log(username);
```
Comment (ol3gle) [inline www/pages/components/file_view.vue.js:727]: Remove log

File: www/pages/components/file_view.vue.js [hunk 2]
```diff
@@ -281,7 +289,18 @@ var appFileView = Vue.component("app-file-view", {
...
-		doShuffle() {
+		copyCurl() {
+            var l = window.location;
+            var filesurl = l.protocol + "//" + l.hostname;
+            if (l.port != "" && (l.port != 443 && l.port != 80)) {
+                    filesurl += ":" + l.port;
+            }
+            filesurl += escape("/api/v1/files");
+            this.curlCommand = "curl -X POST -H \"Authorization: "   + localStorage.Authorization + "\" -F \"file=@path/to/file\" ";
+            this.curlCommand = this.curlCommand + " " + filesurl;
```
Comment (ol3gle) [inline www/pages/components/file_view.vue.js:300]: Use string format

File: www/pages/components/file_view.vue.js [hunk 3]
```diff
@@ -281,7 +289,18 @@ var appFileView = Vue.component("app-file-view", {
         }
     },
 	methods: {
-		doShuffle() {
+		copyCurl() {
+            var l = window.location;
+            var filesurl = l.protocol + "//" + l.hostname;
+            if (l.port != "" && (l.port != 443 && l.port != 80)) {
+                    filesurl += ":" + l.port;
+            }
```
Comment (ol3gle) [inline www/pages/components/file_view.vue.js:297]: What is this check for?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,54 @@
...
+<p align="center">
+  <img alt="uploadCurl" src="media/upload-via-curl.gif" height="500" />
+</p>
+
+* When uploading via command line utility you can specify the mime-type via special header **x-pwndrop-content-type**:
+```
+curl -X POST -H "x-pwndrop-content-type: application/javascript" -H "Authorization: [redacted]" -F "file=@Malicious.exe"  https://<your-server>/api/v1/files
+```
+
+
```
Comment (ol3gle) [inline README.md:25]: Why not in changelog

**Review by ol3gle — COMMENTED** [review]
Some questions

File: README.md (hunk 4 above)
Comment (B1t0n) [reply README.md:25]: It's there as well, but I want also to point out the changes we made on the landing page of this repo.

File: www/pages/components/file_view.vue.js (hunk 3 above)
Comment (B1t0n) [reply www/pages/components/file_view.vue.js:297]: This helps to check if your pwndrop has different port, as part of the cURL command builder

**Pull request merged**